qa/L0_e2e/test_model.py

```diff
@@ -66,6 +67,16 @@ def valid_shm_modes():
     return tuple(modes)
 
 
+# TODO(hcho3): Remove once we fix the flakiness of CUDA shared mem
```

Comment:
Could we link explicitly to the corresponding Triton Github issue?

Reply:
It took me a while to craft a minimal reproducer: https://github.com/triton-inference-server/server/issues/7688. Let me link it now.